### PR TITLE
Filter placeholder

### PR DESCRIPTION
**Pull request by hstandaert**
## Description

screenshot 2018-12-27 at 15 20 01

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [x] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="filter-placeholder box-no-icon">
+  <p>{{ description }}</p>
+
+  <span class="spotimage"></span>
```
Comment (meirege) [inline styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig:4]: @delrueba Is this okay for accessibility? This image is now invisible to screen readers... not sure if that's an issue here...?

File: templates/core/block/block--search-help.html.twig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<div class="filter-placeholder box-no-icon">
+  <p>{{ 'As soon as you start searching, you can filter the results of your search here.'|t }}</p>
+
+  <span class="spotimage"></span>
```
Comment (meirege) [inline templates/core/block/block--search-help.html.twig:4]: Same as above.

File: styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig (hunk 1 above)
Comment (delrueba) [reply styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig:4]: Sure, it's decorative. 
"Abstract blue lightbulb" doesn't add much information.

**Review by delrueba — APPROVED** [review]

File: styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig (hunk 1 above)
Comment (meirege) [reply styleguide/components/31-molecules/filter-placeholder/filter-placeholder.twig:4]: Okay, thought so, but just checking :)

**Review by meirege — APPROVED** [review]

**Pull request merged**